server/bytestream/BUILD

```diff
@@ -2,10 +2,14 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 
 go_library(
     name = "bytestream",
-    srcs = ["bytestream.go"],
+    srcs = [
+        "bytestream.go",
+        "zip.go",
```

Comment:
+1 to this suggestion

Reply:
Sure, branched this out and cleaned it up a bit so that there isn't as much bleeding over between the two packages.  Feels like this could still be a bit better, open to suggestions.